openpype/hosts/maya/api/pipeline.py

```diff
@@ -63,9 +63,11 @@ def __init__(self):
         self._op_events = {}
 
     def install(self):
-        project_name = os.getenv("AVALON_PROJECT")
+        project_name = legacy_io.active_project()
         project_settings = get_project_settings(project_name)
         # process path mapping
+        project_name = legacy_io.active_project()
```

Comment:
I know this is merged - but note: this line is redundant - duplicate code from above.

Reply:
Added #3832